scripts/build-contract

```diff
@@ -8,14 +8,14 @@
 set -eu
 
 build_mode=${BUILD_MODE:-Release}
-cargo_contract_version=${CARGO_CONTRACT_VERSION:-2.0.2}
-rust_toolchain=${RUST_TOOLCHAIN:-1.66.1-x86_64-unknown-linux-gnu}
+cargo_contract_version=${CARGO_CONTRACT_VERSION:-2.2.0}
```

Comment:
I'm not sure if we want to be doing this — we need to ensure that anyone using the verification service from contract version 2.0.2 onwards is able to use it.

Reply:
if users want to using the older version, then they can use the env var to do so.